app/scripts/controllers/permissions/background-api.test.js

```diff
@@ -649,7 +595,7 @@ describe('permission background API methods', () => {
       );
     });
 
-    it('calls updateCaveat with the chain added', () => {
+    it('calls updateCaveat with the chain added and all eip155 accounts synced', () => {
```

Comment:
What does "all eip155 accounts synced" mean?

Reply:
From the eip155 scopeObjects, gather the unique list of eip155 accounts, and then assign that list back to each eip155 scopeObject (overwriting any accounts array value that was previous there)